src/tasks/FilterFeedTask.ts

```diff
@@ -78,6 +82,29 @@ export default class FilterFeedTask extends Task {
 			return;
 		}
 
+		try {
+			const ticketKeys = Array.from( this.knownTickets );
+			const previousTicketResults = await this.jira.search.search( {
+				jql: `${ this.jqlRemoved } AND key in (${ ticketKeys.join( ',' ) })`,
+				fields: ['key'],
+			} );
+
+			if ( !previousTicketResults.issues ) {
+				FilterFeedTask.logger.error( 'Error: no issues returned by JIRA' );
+				return;
```

Comment:
What if the query actually doesn't return any issues, and not because of an error? I can see this happen, for example if the bot just got started, and a ticket got fixed. Then, this query would return no issues (because the fixed ticket isn't unresolved), and this returns. Therefore the ticket doesn't get added to the known tickets list, and the ticket gets posted over and over again.

Reply:
Generally, any errors when trying to determine whether any ticket has been reopened shouldn't return here, because otherwise the same thing would happen; the newly fixed tickets wouldn't be added to the known tickets list.

Reply:
It might make sense to extract this into separate methods, too.

Reply:
Ok, It's been a few months but I finally got around to removing the return 😆